papers/bibtex.py

```diff
@@ -35,6 +35,22 @@
 # Others is a reserved Bibtex keyword
 OTHERS_RE = re.compile(r'( and )?\s*others\.?\s*$', re.IGNORECASE | re.UNICODE)
 
+PAPER_TYPE_TO_BIBTEX = [
+   ('journal-article', 'article'),
+   ('proceedings-article', 'inproceedings'),
+   ('book-chapter', 'book'),
```

Comment:
How about using the `incollection` type for book chapters?

Reply:
:+1: will fix